src/ripple/protocol/impl/AccountID.cpp

```diff
@@ -143,11 +103,8 @@ noAccount()
 bool
 to_issuer(AccountID& issuer, std::string const& s)
 {
-    if (s.size() == (160 / 4))
-    {
-        issuer.SetHex(s);
+    if (issuer.parseHex(s))
```

Comment:
Breaks backward compatibility

Reply:
Kinda: so, the code previously checked for `s.size() == (160/4)` so the only case where this would break is if someone passed in a string with the right length but which included a non-hex character: the previous code would detect that as the end of the hex string. After processing all the hex input the function would execute: `return !*pEnd;` which should duplicate the semantics of `SetHexExact`: if `*pEnd` is a `NUL` character it returns `true`, othrwise it returns `false`.